test/Rules/BUY_PROPERTY.test.js

```diff
@@ -0,0 +1,84 @@
+const expect = require('chai').expect;
+const EventEmitter = require('events');
+const sinon = require('sinon');
+const mockUIFactory = require('../mocks/UI');
+
+const { GameState } = require('../../entities/GameState');
+const { createPlayer } = require('../testutils');
+const config = require('../../config/monopolyConfiguration');
+const WealthService = require('../../entities/WealthService');
+const { cloneDeep } = require('lodash');
+
+describe('Rules -> BUY_PROPERTY', () => {
+  let gameState;
+  let userInterface;
+  let eventBus;
+  const RULES = require('../../entities/Rules');
+  const ORIENTAL_AVENUE_PROPERTY = 2;
```

Comment:
I'm not saying this is racist, but I do think this should be `TEST_PROPERTY` instead because the properties are configurable, so this shouldn't be named to reflect a specific property as it might cause cognitive problems later in life (i.e., "Why is 'Oz's Beer Tavern' called 'ORIENTAL_AVENUE_PROPERTY' ?" _Keegan proceeds to changing the const name to OZS_BEER_TAVERN ... more confusion ensues..._)